Tests/AdaptiveStyleTests.swift

```diff
@@ -143,6 +143,9 @@ class AdaptiveStyleTests: XCTestCase {
             (.numberCase(.lower), false),
             (.numberSpacing(.monospaced), false),
             (.numberSpacing(.proportional), false),
+            (.fractions(.disabled), true),
+            (.fractions(.diagonal), false),
+            (.fractions(.vertical), false),
```

Comment:
#req this causes the test to fail. This array should contain only adjustments of features that EBGaramond12-Regular supports, so delete the `.vertical` line.